hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/TableSnapshotInputFormatImpl.java

```diff
@@ -128,6 +129,12 @@
 
   public static final boolean SNAPSHOT_INPUTFORMAT_SCAN_METRICS_ENABLED_DEFAULT = true;
 
+  /**
+   * The {@link ReadType} which should be set on the {@link Scan} to read the HBase Snapshot, default STREAM.
+   */
+  public static final String SNAPSHOT_INPUTFORMAT_SCANNER_READTYPE = "hbase.TableSnapshotinputFormat.scanner.readtype";
```

Comment:
TableSnapshotinputFormat  =>  TableSnapshotInputFormat  ?

Reply:
hah! Thanks for noticing.